main/meson.build

```diff
@@ -118,7 +119,13 @@ endif
 if dep_gtk4.version() == 'unknown' or dep_gtk4.version().version_compare('>=4.8')
     vala_args += ['-D', 'GTK_4_8']
 endif
-exe_dino = executable('dino', sources, c_args: c_args, vala_args: vala_args, dependencies: dependencies, install: true)
+
+link_args = []
+if host_machine.system() == 'windows'
+  link_args += ['-Wl,--export-all-symbols']
```

Comment:
There is such code in `main\CMakeLists.txt`:
```cmake
if(WIN32)
    add_link_options("-Wl,--export-all-symbols")
    set(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> --use-temp-file -O coff <DEFINES> -i <SOURCE> -o <OBJECT>")
    add_executable(dino ${MAIN_VALA_C} ${MAIN_GRESOURCES_TARGET} dino-info.rc)
else(WIN32)
```
It calls the RC compiler to compile Windows resources, including adding an icon, I think this should be added to the Meson build.

Reply:
fixed.